omou-frontend/src/components/FeatureViews/Scheduler/SessionView.js

```diff
@@ -0,0 +1,185 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as calenderActions from '../../../actions/calenderActions';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import BackButton from "../../BackButton.js";
+import SessionActions from "./SessionActions";
+import '../../../theme/theme.scss';
+import './scheduler.scss'
+
+//Material UI Imports
+import Grid from "@material-ui/core/Grid";
+import Paper from "@material-ui/core/Paper";
+import { Divider, LinearProgress, Typography, ListItem } from "@material-ui/core";
+import List from '@material-ui/core/List'
+import ListItemText from '@material-ui/core/ListItemText'
+
+
+
+import NewCourse from "@material-ui/icons/CalendarToday";
+import AssignmentIcon from "@material-ui/icons/Assignment";
+import UpdateTeacher from "@material-ui/icons/PersonAdd";
+import EditIcon from "@material-ui/icons/Edit";
+import CalendarIcon from "@material-ui/icons/CalendarTodayRounded";
+import Button from "@material-ui/core/Button";
+
+
+
+class SessionView extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {}
+    }
+
+    componentWillMount() {
+        this.setState(() => {
+
+            let sessionData = this.props.courseSessions[this.props.computedMatch.params.course_id][this.props.computedMatch.params.session_id]
+            let courseData = this.props.courses[this.props.computedMatch.params.course_id]
+            console.log("Session data: ", sessionData)
+            console.log("courseData: ", courseData)
+
+            return {
+                sessionData: sessionData,
+                courseData: courseData
+            }
+        })
+
+
+    }
+
+    render() {
+        console.log(this.state)
+
+        const flexContainer = {
+            display: "flex",
+            justifyContent: "flex-start",
+            width: "50%",
+            padding: 0,
+        };
+
+
+
+        return (
+            <Grid className="">
+                <Paper className={"paper"}>
+                    <SessionActions />
+                </Paper>
+                <Grid className={'main-session-container'}>
+                    <Paper className={'paper'}>
+                        <Grid className={'session-button'}>
+                            <BackButton />
+                        </Grid>
+                        <Divider />
+
+                        <Grid item xs={12}>
+                            <Typography className={'session-view-title'} align={'left'} variant={'h3'}>
+                                {this.state.courseData.title}
+                            </Typography>
+                        </Grid>
+                        <List style={flexContainer}>
+                            <ListItem>
+                                <ListItemText
+                                    primary="Subject"
+                                    secondary={this.state.courseData.subject} />
```

Comment:
What is secondary text used for? if it's what I think it is, you should use the <Typography> component where you can specify text styles. for example:  
```
<Grid item md={12}>
    <Typography variant = "h2"> Subject </Typography>
    <Typography varient = "p">{this.state.courseData.subject} </Typography>
</Grid>
```

Reply:
Secondary texted is used for text below the primary.
So in any instance it will say the subject of the session like 
Subject
Math

You don't need to mess with any css to get the words aligned. 
If you go to https://material-ui.com/components/lists/#lists > interactive. You can see how the secondary works.

Reply:
Actually you way seems easier to style the words. I'll make the changes.